[3.2] docs(gantt): Revamped Overview and Gantt Splitter position para… (#880)
telerik/blazor-docs: components/gantt/timeline/templates/overview.md

# Templates for the Gantt Timeline

> Customize the appearance of the Gantt Timeline through the templates it provides, so you can add more details for your users and have full control over how they are rendered.

Reviewer: Split long sentences.

```suggestion
Customize the appearance of the Gantt Timeline through its templates. They will enable you to add more details for the users and have full control over the rendering.
```
